WordPress/Classes/ViewRelated/Post/PostListFilterSettings.swift

```diff
@@ -0,0 +1,166 @@
+import Foundation
+import WordPressComAnalytics
+
+@objc class PostListFilterSettings: NSObject {
+    private static let currentPostAuthorFilterKey = "CurrentPostAuthorFilterKey"
+    private static let currentPageListStatusFilterKey = "CurrentPageListStatusFilterKey"
+    private static let currentPostListStatusFilterKey = "CurrentPostListStatusFilterKey"
+
+    let blog:Blog
```

Comment:
Add a space after the colon (since this is how we do it throughout most of our codebase).

`let blog: Blog`

Same thing for the 2 lines below.